docs/en/observability/application-and-service-monitoring.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[application-and-service-monitoring]]
+= Application and service monitoring
+
+++++
+<titleabbrev>Applications and services</titleabbrev>
+++++
+
+Explore the topics in this section to learn how to observe and monitor software applications and services running in your environment.
+
+[cols="1,1"]
+|===
+|<<apm>>
+|Monitor software services and applications in real time, by collecting detailed performance information on response time for incoming requests, database queries, calls to caches, external HTTP requests, and more.
+
+|<<monitor-java-app>>
+|Monitor a Java application using Elastic Observability: Logs, Infrastructure metrics, APM, and Uptime.
```

Comment:
This feels strange to me since (1) it's the only language-specific item at this level and (2) we also include content related to Java applications throughout the APM section. That said, I don't have an alternative to suggest so 🤷 .

Reply:
Yah I feel the same. It's really an outlier. I wonder if we event want to continue publishing it since the info is so old. Maybe one option is to hide it from the TOC. I don't have a good answer to this problem.

Reply:
Going with the option of flagging this as a tutorial and moving it to the end of the section.